gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/IndexGigawordWithElasticSearch.java

```diff
@@ -38,8 +38,9 @@
           "\tindexName: the name of the index in a running Elastic Search server to add the documents to\n" +
           "\tcorpusDirectoryPath: the path to a directory where corpus (i.e. LDC2011T07 English Gigaword 5th edition)\n" +
           "\t\thas been extracted. \n" +
-          "\t\tIf indexing Gigaword, make sure the version is the plain Gigaword (LDC2011T07) file and not the other versions.\n" +
           "\tformat: LTF or gigaword" +
+          "\tcorpusFormat: flag whether the version if Gigaword is plain (\"gigaword\") or " +
```

Comment:
I don't think we want two separate parameters - just make `annotated_gigaword` another option for `format`

Reply:
Fixed in latest commit.

Reply:
@az79nefy : The code is updated but the usage message still has two separate parameters.